mlflow/store/tracking/file_store.py

```diff
@@ -221,7 +221,7 @@ def _get_active_experiments(self, full_path=False):
     def _get_deleted_experiments(self, full_path=False):
         return list_subdirs(self.trash_folder, full_path)
 
-    def list_experiments(self, view_type=ViewType.ACTIVE_ONLY):
+    def list_experiments(self, view_type=ViewType.ACTIVE_ONLY, max_results=None, page_token=None):
```

Comment:
Interestingly, in the three or four other pagination PRs that have gone in, file_store has remained untouched - that's why I hadn't added anything along those lines (a. because I wasn't sure that we wanted it since we didn't for anything else, b. because I didn't have a template to work from making me less confident). I've added pagination here using `_search_runs` as a model, but it sounds like file_store in general needs some discussion.

Reply:
Ah @wamartin-aml yeah sorry for the confusion - the other APIs we've most-recently added pagination to are model registry APIs, which don't have file_store support (i.e. in order to use the model registry you need to use a SQL-backed store). In this case, we do need to add it for FileStore since ``list_experiments`` is supported in the FileStore

Reply:
Thanks for doing so!